pandas/formats/format.py

```diff
@@ -2138,6 +2016,31 @@ def formatter(value):
 
         return formatter
 
+    def _get_formatted_data(self):
+        """
+        Returns formatted float values. This is different from
```

Comment:
these entire module is internal - in reality none should be leading _

we have very inconsistent usage